lib/server.js

```diff
@@ -38,6 +34,21 @@ var TEMPLATE = ['<script id="%s" type="application/javascript">var ',
 
 exports.create = function create(createOptions) {
 
+  // safely require the peer-dependencies
+  var React = util.safeRequire('react');
+
+  try {
+    var Router = require('react-router');
+  } catch (err) {
+    if (!Router && createOptions.routes)
+      throw new Error('React Router is required as a peer dependency. Run `npm instal react-router` to install React Router.');
+    }
+  }
+
+  if (!React) {
+    throw new Error('React is required as a peer dependency. Run `npm instal react` to install react.');
```

Comment:
Ok, I don't think we need to throw a new Error then. Just throw the caught error.